### PR TITLE
Bump rubygem-fog-ovirt to 2.0.0-1

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/foreman/rubygem-fog-ovirt/rubygem-fog-ovirt.spec [hunk 1]
```diff
@@ -86,12 +89,17 @@ cp -pa .%{gem_dir}/* \
 %doc %{gem_instdir}/CONTRIBUTING.md
 %doc %{gem_instdir}/CONTRIBUTORS.md
 %{gem_instdir}/Gemfile
+%{gem_instdir}/Gemfile.local.rb
```
Comment (tbrisker) [inline packages/foreman/rubygem-fog-ovirt/rubygem-fog-ovirt.spec:92]: This one should be `%exclude`, it was added by mistake and will be removed in the next release.

File: packages/foreman/rubygem-fog-ovirt/rubygem-fog-ovirt.spec [hunk 2]
```diff
@@ -1,32 +1,36 @@
...
-Requires: %{?scl_prefix}rubygem(fog-core)
+Requires: %{?scl_prefix}rubygem(activesupport)
 Requires: %{?scl_prefix}rubygem(fog-core)
 Requires: %{?scl_prefix}rubygem(fog-json)
 Requires: %{?scl_prefix}rubygem(fog-xml)
-Requires: %{?scl_prefix}rubygem(fog-xml)
-Requires: %{?scl_prefix}rubygem(ovirt-engine-sdk) >= 4.1.3
+Requires: %{?scl_prefix}rubygem(ovirt-engine-sdk) >= 4.3.1
 Requires: %{?scl_prefix}rubygem(rbovirt) >= 0.1.5
 Requires: %{?scl_prefix}rubygem(rbovirt) < 0.2
```
Comment (tbrisker) [inline packages/foreman/rubygem-fog-ovirt/rubygem-fog-ovirt.spec:27]: this should be dropped but it's still in the gemspec so ok for now

**Review by ekohl — APPROVED** [review]

File: packages/foreman/rubygem-fog-ovirt/rubygem-fog-ovirt.spec [hunk 3]
```diff
@@ -1,32 +1,36 @@
...
+Requires: %{?scl_prefix_ruby}ruby >= 2.0.0
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(fog-core)
+Requires: %{?scl_prefix}rubygem(activesupport)
 Requires: %{?scl_prefix}rubygem(fog-core)
 Requires: %{?scl_prefix}rubygem(fog-json)
 Requires: %{?scl_prefix}rubygem(fog-xml)
-Requires: %{?scl_prefix}rubygem(fog-xml)
-Requires: %{?scl_prefix}rubygem(ovirt-engine-sdk) >= 4.1.3
+Requires: %{?scl_prefix}rubygem(ovirt-engine-sdk) >= 4.3.1
```
Comment (ekohl) [inline packages/foreman/rubygem-fog-ovirt/rubygem-fog-ovirt.spec:25]: Repoclosure is unhappy about this

Comment (evgeni) [reply packages/foreman/rubygem-fog-ovirt/rubygem-fog-ovirt.spec:25]: and rightfully so, updated to include the bumped sdk gem

**Review by tbrisker — APPROVED** [review]

**Pull request merged**